libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPromptSettings.cs

```diff
@@ -40,5 +40,13 @@ public class OAuthPromptSettings
         /// </summary>
         /// <value>The number of milliseconds the prompt waits for the user to authenticate.</value>
         public int? Timeout { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the <see cref="OAuthPrompt"/> should end upon
+        /// receiving an invalid message.

```

Comment:
Could you more detail here - what's an invalid message, and why would folks want to typically set this to true?

Reply:
addressed